JobConfig/beam/epilog.fcl

```diff
@@ -0,0 +1,6 @@
+#turn off MCTrajectories
+physics.producers.g4run.TrajectoryControl.produce : false
```

Comment:
This seems to force off MCTrajectories in ALL beam jobs without any way to enable them for an individual fcl file.    Please default MCTrajectory creation to false in prolog instead and remove  this epilog setting.

Reply:
Removed.